test/unit/core/selectors.ts

```diff
@@ -0,0 +1,24 @@
+import { Selectors as FluxSelectors, Store } from '@storefront/flux-capacitor';
+import ProductTransformer from '../../../src/core/product-transformer';
+import Selectors from '../../../src/core/selectors';
+import suite from '../_suite';
+
+suite('Selectors', ({ expect, spy, stub }) => {
+  describe('transformProducts', () => {
+    it('should wrap around productTransformer', () => {
+      const products = { e: 5 };
+      const transformer = spy((x) => x);
+      const state = { b: 2 };
+      const structure = { c: 3 };
+      const configSelector = stub(FluxSelectors, 'config').returns({ structure });
+      const productTransformer = stub(ProductTransformer, 'transformer').returns(transformer);
+
+      const transformedProducts = Selectors.transformProducts(<any>state, <any>products);
```

Comment:
rather than casting to `<any>` here, just add `any` as the type when declaring `products` and `state`